### PR TITLE
Logo + Color Scheme

### PR DESCRIPTION
**Pull request by sloan-seamus**
Modified color scheme.	
- Ability to change to dark mode eventually

Changed the logo to text

### CODE REVIEW

File: frontend/insurance/src/app/modules/home/home.component.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.view-button {
```
Comment (dsifford) [inline frontend/insurance/src/app/modules/home/home.component.scss:1]: display flex?

File: frontend/insurance/src/sass/spacing/spacing.scss [hunk 2]
```diff
@@ -1,72 +1,79 @@
...
-  .pbl {
-    padding-bottom: 16px !important;
-  }
-  
-  .pbh {
-    padding-bottom: 32px !important;
-  }
-  
\ No newline at end of file
+  margin-left: 4px !important;
```
Comment (dsifford) [inline frontend/insurance/src/sass/spacing/spacing.scss:2]: All the `!important`s here make me want to cry

File: frontend/insurance/src/app/core/components/navigation-bar/navigation-bar.component.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
+//Keeping in case we want to switch to logos later
```
Comment (ej3076) [inline frontend/insurance/src/app/core/components/navigation-bar/navigation-bar.component.scss:1]: make sure to run `ng lint` to make sure there are no errors that show up. this should throw an error to the linter.

File: frontend/insurance/src/sass/spacing/spacing.scss (hunk 2 above)
Comment (ej3076) [inline frontend/insurance/src/sass/spacing/spacing.scss]: What happened here and why did you change the class name? Did you fix all references to this class?

**Review by ej3076 — APPROVED** [review]
Run `ng lint`

One question on a class name change.

File: frontend/insurance/src/sass/spacing/spacing.scss (hunk 2 above)
Comment (sloan-seamus) [reply frontend/insurance/src/sass/spacing/spacing.scss:2]: Sometimes it be like that.
